src/admin-views/troubleshooting/ea-status/current-status.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * View: Troubleshooting - EA Status Table Current Status.
+ *
+ * @since TBD
+ *
+ *
+ * @param array<string|string> $status_icons An array of icons for the EA Status table.
+ * @param \Tribe__Main         $main         An instance of the main class of Tribe Common.
+ *
+ */
+
+$icon     = 'success';
+$notes    = '&nbsp;';
+$message  = esc_html_x( 'Imports Enabled in Settings', '', 'tribe-common' );
+$disabled = tribe_get_option( 'tribe_aggregator_disable', false );
+
+if ( $disabled ) {
+	$icon         = 'error';
+	$message      = _x( 'Imports disabled in Settings', '', 'tribe-common' );
+	$settings_url = Tribe__Settings::instance()->get_url( array( 'tab' => 'imports' ) );
+	$notes 		  = sprintf(
+		'<a href="%1$s">%2$s</a>',
+		$settings_url,
```

Comment:
Might want to escape
```suggestion
		esc_url( $settings_url ),
```